app/models/accounting/bank.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Accounting
+  class Bank < ApplicationRecord
+    # Validates that the city/country code provided is in the format: USA, NYC, etc.
+    CITY_COUNTRY_CODE_REGEX = /\A[a-zA-Z]+\z/
+
+    # Validates the swift code is in the format: UNCRITMMXXX
+    SWIFT_CODE_REGEX = /\A[a-zA-Z0-9]+\z/
+
+    # Validates the url provided is in the format: https://example.com | http://example.com
+    URL_REGEX = %r{\Ahttps?:\/\/(?:[a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}(?:\/.*)?\z} # rubocop:disable Style/RedundantRegexpEscape
+
+    validates :name, :country_code, :city_code, :routing_number, :swift_code, :website,
+              presence: true
+    validates :country_code, :city_code, :swift_code, uniqueness: { case_sensitive: true }
```

Comment:
Please remove `uniqueness` with case sensitive, because this will lead for the app to only be able to save one bank per country since in the column `country_code` values can only be unique.